site/content/guides/openshift-load-balancing.adoc

```diff
@@ -20,7 +20,7 @@ v1.0, 2015-10-30
 
 include::site/layouts/variables.adoc[]
 
-Interating with a Load Balancing scheme to ensure high availability is mantained to the cluster components 
+This guide will walk you how to leverage Load Balancers for cluster and application high availability.
```

Comment:
`walk you THROUGH`

Reply:
should be fixed already, I see it in the file.